golang/example/example_lambda_configurator.go

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2020 - present Kurtosis Technologies LLC.
+ * All Rights Reserved.
+ */
+
+package example
+
+import (
+	"encoding/json"
+	"github.com/kurtosis-tech/kurtosis-lambda-api-lib/golang/lib/lambda"
+	"github.com/palantir/stacktrace"
+	"github.com/sirupsen/logrus"
+)
+
+type ExampleLambdaConfigurator struct{}
+
+func NewExampleLambdaConfigurator() *ExampleLambdaConfigurator {
+	return &ExampleLambdaConfigurator{}
+}
+
+func (t ExampleLambdaConfigurator) SetLogLevel(logLevelStr string) error {
```

Comment:
Since loglevel isn't going to be a formal concept from Kurtosis' perspective, we actually don't need the `SetLogLevel` function and this code should be moved inside the `ParseParamsAndCreateLambda` function

Reply:
EDIT: Actually, this function isn't even in the LambdaConfigurator interface - def can be deleted

Reply:
Yes, when I started creating it I put this method, but then I replaced it using the private method but I forgot to remove this one. So good catch!